src/inspector_js_api.cc

```diff
@@ -261,7 +261,7 @@ void Url(const FunctionCallbackInfo<Value>& args) {
   Agent* agent = env->inspector_agent();
   InspectorIo* io = agent->io();
 
-  if (!io) return;
+  if (!io || !io->publish_uid_binding()) return;
```

Comment:
Is this technically a breaking change?

Reply:
as long as I passed "binding" by default - it is not a breaking change but I removed binding from this PR.